end-to-end/package-jazzer-js.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+cd ..
+npm install
+npm run build
+npm run build --workspace='@jazzer.js/fuzzer'
```

Comment:
Time to combine the two build commands?

Reply:
I don't know enough about node to know why it's 2 commands currently but I think it should be a separate PR anyway.